_src/pattern-library/sass/components/_layouts.scss

```diff
@@ -0,0 +1,700 @@
+// ------------------------------
+// edX Pattern Library: Components - Layouts
+
+// About: Contains base settings and options for the preset default Pattern Library UI Layouts.
+// ----------------------------
+
+// #CONFIG
+// #GLOBALS
+// #UTILITY
+// #CANNED - IN VIEW
+// #CANNED - VIEW
+
+// ----------------------------
+
+// ----------------------------
+// #CONFIG
+// ----------------------------
+$susy-flush:            layout($susy-default inside); // set susy to make columns flush
+$susy-reversed:         layout($susy-default $grid-direction-reversed); // set susy to make columns display in reverse order
+$susy-view:             layout($susy-default auto); // set susy to make columns view display in reverse order
+$susy-view-flush:       layout($susy-default auto inside); // set susy to make view columns display in reverse order
+$susy-view-reversed:    layout($susy-default auto $grid-direction-reversed); // set susy to make view columns display in reverse order
+
+// ----------------------------
+// #GLOBALS
+// ----------------------------
+
+
+// ----------------------------
+// #UTILITY
+// ----------------------------
+
+// CASE: Layout with gutter-separated columns
+%layout-default {
+    @include container($susy-default);
+}
+
+// CASE: Layout with flush columns
+%layout-flush {
+    @include container($susy-flush);
+}
+
+// CASE: Layout with reversed columns
+%layout-reversed {
+    @include container($susy-reversed);
+}
+
+// CASE: View-based layout
+%layout-view {
+    @include container($susy-view);
+}
+
+// CASE: View-based layout with flush columns
+%layout-view-flush {
+    @include container($susy-view-flush);
+}
+
+// CASE: View-based layout with reversed columns
+%layout-view-reversed {
+    @include container($susy-view-reversed);
+}
+
+
+// basic layout column
+%layout-column {
+    @include span(full); // plan for mobile first
+}
+
+// ----------------------------
+// #CANNED - IN VIEW
+// ----------------------------
+
+// basic layout
+.layout {
+    @extend %layout-default;
+}
+
+// basic layout - flush
+.layout-flush {
+    @extend %layout-flush;
+}
+
+// basic columns
+.layout-col {
+    @extend %layout-column;
+}
+
+// Full Layout
+.layout-full {
+    @extend %layout-default;
+
+    .layout-col-a,
+    .layout-col-b,
+    .layout-col-c,
+    .layout-col-d {
+        @include span(full);
+    }
+}
+
+// 1/4 + 3/4 Layout
+.layout-1q3q {
+
+    // CASE: standard layout
+    &.layout {
+        @extend %layout-default;
+        @include susy-breakpoint($bp-screen-md, $susy-default) {
```

Comment:
Note on syntax here, we're using 1 of 2 in Susy ways to call a media query that's managed by Breakpoint Sass. This way lets us pass in a second argument that tells Susy what layout rules to apply to the rules inside of the query (this is useful for when we change layout settings from default to flush to reverse).

To read more see: http://www.zell-weekeat.com/susy2-tutorial-2/ (good explanation on the 2 Breakpoint methods) and http://susy.readthedocs.org/en/latest/toolkit/#breakpoint